airflow/provider_info.schema.json

```
@@ -16,11 +16,33 @@
     },
     "hook-class-names": {
       "type": "array",
-      "description": "Hook class names that provide connection types to core",
+      "description": "Hook class names that provide connection types to core((deprecated by connection-types)",
       "items": {
         "type": "string"
+      },
+      "deprecated": {
+        "description": "The hook-class-names property has been deprecated in favour of connection-types which is more performant version allowing to only import individual Hooks rather than all hooks at once",
+        "deprecatedVersion": "2.2"
```

```suggestion
        "deprecatedVersion": "2.2.0"
```

Probably to be more specific, wdyt?

---

Yep.